dockers/docker-sonic-telemetry/Dockerfile.j2

```diff
@@ -26,6 +27,10 @@ COPY ["start.sh", "telemetry.sh", "dialout.sh", "/usr/bin/"]
 COPY ["telemetry_vars.j2", "/usr/share/sonic/templates/"]
 COPY ["supervisord.conf", "/etc/supervisor/conf.d/"]
 COPY ["files/supervisor-proc-exit-listener", "/usr/bin"]
+COPY ["files/container_monitor.py", "/usr/bin"]
 COPY ["critical_processes", "/etc/supervisor"]
 
+# ingrain version into start.sh
+RUN [ -f /usr/bin/start.sh ] && sed -ri "s/^IMAGE_VERSION.*$/IMAGE_VERSION=$image_version/;" /usr/bin/start.sh
```

Comment:
if you have build the image_version in the env during docker build, why not directly use the image_version env in the start.sh? why need to do the sed?

Reply:
Good point. I did not know this possibility. Sure.

Reply:
done